### DISCUSSION TITLE
Numpy

### DISCUSSION THREAD

**Discussion by mssanamateen** [mssanamateen/Artificial-Intelligence-Lab#21] [Q&A] [answered]
Why do we use numpy? How to check latest version of numpy?

**Comment by Divk-Ashwin** [comment, marked as answer]
->Abbreviation of Numpy is numerical python, It is majorly used for advanced mathematical calculations. The main advantages numpy has is the performance it offers, When you need to do the same operation on a lot of different numbers, it can be a lot faster if you know you're doing them in bulk rather than treating it as a series of individual one-time operations.
->In jupyter notebook, We can install  numpy by using command:  pip install numpy
->To check the latest version of numpy we use command: np.__version__
->The latest version of numpy running is: '1.26.4'
